tests/library/steps.py

```diff
@@ -165,7 +154,14 @@ def delete_all_data(client):
 
 @allure.step('Create hostservice in specific cluster')
 def create_hostcomponent_in_cluster(client, cluster, host, service, component):
-    return client.cluster.hostcomponent.create(cluster_id=cluster['id'],
-                                               hc=[{"host_id": host['id'],
-                                                    "service_id": service['id'],
-                                                    "component_id": component['id']}])
+    hc = [{"host_id": host['id'],
+           "service_id": service['id'],
+           "component_id": component['id']}]
+    return client.cluster.hostcomponent.create(cluster_id=cluster['id'], hc=hc)
+
+
+@allure.step('Check if action {action_name} state is {state_expected}')
+def check_state(action_name, state_current, state_expected):
```

Comment:
```suggestion
def check_action_state(action_name, state_current, state_expected):
```
Or make step title more general by removing "action" word

Reply:
+ add typing to args

Reply:
done